examples/2d/texture_atlas.rs

```diff
@@ -51,7 +51,11 @@ fn setup(
     let mut texture_atlas_builder = TextureAtlasBuilder::default();
     for handle in &rpg_sprite_handles.handles {
         let handle = handle.typed_weak();
-        let texture = textures.get(&handle).expect("Textures folder contained a file which way matched by a loader which did not create an `Image` asset");
+        let Some(texture) = textures.get(&handle) else {
+            warn!("{:?} did not resolve to an `Image` asset.", asset_server.get_handle_path(handle));
```

Comment:
This downgrades the issue from a panic to a warning, yet the next line will try to add the non-existent texture. 

Any reason for this downgrade? Does it work?

Reply:
It works. The next line is `continue` so the non-texture is never added to the atlas.

My intent was to fix the super confusing error message, and provide the user with the path to the asset that is causing problems.

I am not particularly fussed about panicking vs warning but I am imagining things like stray `.DS_Store` files or something triggering this and it doesn't seem worth panicking about to me.

Reply:
That seems reasonable, thanks for the explanation 😄